### DISCUSSION TITLE
How do I install the devkit? I'm confused on where to place everything.

### DISCUSSION THREAD

**Discussion by olifurz** [CosmosOS/Cosmos#2097] [Q&A]
Hi,

I've got the folders I need which I downloaded off the github pages linked to the devkit install documentation. I'm just really confused where to place everything. Please help! I have ran installvs-2019.bat and I have just gotten an "unable to locate" error.
![image](https://user-images.githubusercontent.com/97028115/152827707-899878a0-8d06-403d-bb15-470ab5d276c7.png)

**Comment by Mantaraix** [comment]
Staging should go like this:


CosmosMain/
├── Cosmos/
│   ├── (contents of Cosmos-master.zip)
├── Common/
│   ├── (contents of Common-master.zip)
├── XSharp/
│    ├── (contents of XSharp-master.zip)
└── IL2CPU/
      ├── (contents of IL2CPU-master.zip)

**Reply by olifurz** [reply]
Where is CosmosMain though?

**Reply by Mantaraix** [reply]
That's just a parent folder, you can create and name that to whatever you prefer

**Reply by olifurz** [reply]
![image](https://user-images.githubusercontent.com/97028115/152867602-8d7c640b-f05a-43cb-bf86-6f9aa7582dbc.png)
![image](https://user-images.githubusercontent.com/97028115/152867746-88924ab4-12c0-4cff-9944-7eef3dc20b6e.png)

I get these errors.
